src/cow-react/modules/swap/pure/EthFlow/EthFlowStepper/index.cosmos.tsx

```diff
@@ -350,6 +313,47 @@ const STEPS: Step[] = [
         state: SmartOrderStatus.CREATION_MINED,
         isExpired: true,
       },
+      creation: { ...defaultProps.creation, failed: false },
+    },
+  },
+  {
+    description: '[TX-FAILED-CREATION] Creation tx failed',
+    props: {
+      ...defaultProps,
+      order: {
+        ...defaultOrderProps,
+        state: SmartOrderStatus.CREATING,
+        isExpired: false,
+        rejectedReason: 'Transaction failed',
+      },
+      creation: { ...defaultProps.creation, failed: true },
+    },
```

Comment:
Adding pic for this. In case it helps other reviewers

<img width="1116" alt="image" src="https://user-images.githubusercontent.com/2352112/209008387-a4f9343f-045f-4f2f-bc0d-08e4ec2387c7.png">


It looks good to me!